docs/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+# Troubleshooting Marketplace Operator
+
+This document lists out some common failures related to the Marketplace operator a user might encounter, along with ways to troubleshoot the failures. If you encouter a failure related to the Marketplace operator that is not listed in this document, but should be, please open an issue to have the failure appended to this document. 
+
+
+Table of contents
+===================
+
+1. [No packages show up in the UI](https://github.com/anik120/operator-marketplace/blob/troubleshooting-guide/docs/troubleshooting.md#no-packages-show-up-in-the-ui)
+2. [OperatorSource failing to download from datastore](https://github.com/anik120/operator-marketplace/blob/troubleshooting-guide/docs/troubleshooting.md#operatorsource-failing-to-download-from-datastore)
+3. [OperatorSource `MESSAGE` reads unknown error (status 404)]()
```

Comment:
Links here and other places have been left blank in anticipation for an answer of the question above.

Reply:
You can use the relative path, or in this case use the tag:
`#operatorsource-failing-to-download-from-datastore`